pkg/apis/garden/v1beta1/helper/helpers.go

```diff
@@ -255,7 +290,7 @@ func SetMachineImages(profile *gardenv1beta1.CloudProfile, images []gardenv1beta
 	return nil
 }
 
-// SetMachineImages sets imageVersions to the matching imageName in the machineImages.
+// SetMachineImageVersionsToMachineImage sets imageVersions to the matching imageName in the machineImages.
```

Comment:
Why is this renamed as well?

Reply:
Because I think this name better fits on the functions's behaviour.

Reply:
Just checked the function again, it's fine. Thanks! :)